### DISCUSSION TITLE
ssh authentication doesn't like Windows

### DISCUSSION THREAD

**Discussion by StevePoling** [community/community#38328] [Other Feature Feedback, Questions, & Ideas]
I created a fresh ssh ed25519 key pair. Edited ~/.ssh/config to pick up my id_github.pub and copied it into my github ssh keys. (This is after successfully doing the same on my MacOSX and Ubuntu machines.) I ran
```
ssh -T git@github.com
```
And it authenticates just fine. 

I cloned my repository onto my Windows machine using a HTTP url (maybe HTTPS, I forget). Then changed its remote URL to git@github.com:myuser/myrepo.git. But I cannot git pull or git push. It can't authenticate or can't see the repo.

My github.com settings page says my ssh keys for MacOS and Ubuntu have both been used recently, but the ssh key for my Windows machine have never been used. Is this a fight going with Windows' credential.helper or something? (Yes. I rebooted. It's Windows.) 

What's the next thing to check?

**Comment by ossanna16** [comment]
Hi there @StevePoling and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
